gobblin-service/src/main/java/org/apache/gobblin/service/modules/orchestration/DagManager.java

```diff
@@ -672,7 +672,7 @@ private void cancelDag(DagId dagId) throws ExecutionException, InterruptedExcept
       log.info("Cancel flow with DagId {}", dagToCancel);
       if (this.dagToJobs.containsKey(dagToCancel)) {
         List<DagNode<JobExecutionPlan>> dagNodesToCancel = this.dagToJobs.get(dagToCancel);
-        log.info("Found {} DagNodes to cancel.", dagNodesToCancel.size());
+        log.info("Found {} DagNodes to cancel (DagId {}).", dagNodesToCancel.size(), dagToCancel);
         for (DagNode<JobExecutionPlan> dagNodeToCancel : dagNodesToCancel) {
           cancelDagNode(dagNodeToCancel);
         }
```

Comment:
Is this change intended in this PR? Seems unrelated to Iceberg

Reply:
I added the dagId/correlator from the log line just above to this message.  didn't seem like a stand-alone fix, so I piggy-backed here